### PR TITLE
swagger with authentication

### PR DESCRIPTION
**Pull request by ArikLeikin**
## Task link from JIRA:
link: <https://mssplinter10-1711638164183.atlassian.net/browse/FP-47>

## Description:
{description}

## Code Owners:
{code_owners}

### Summary
_Write an overview_

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -38,6 +47,6 @@ async function bootstrap() {
   app.use(passport.initialize());
   app.use(passport.session());
   app.use(cookieParser());
-  await app.listen(3000);
+  await app.listen(8080);
```
Comment (nitzanto) [inline src/main.ts:50]: Why was this changed? The docker port is 3000

File: src/chat/transcript/transcript.controller.ts [hunk 2]
```diff
@@ -32,6 +70,20 @@ export class TranscriptController {
...
+  })
+  @ApiResponse({
+    status: HttpStatus.UNAUTHORIZED,
+    description: 'Unauthorized',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'ID of the transcript to delete',
+    type: Number,
+  })
```
Comment (nitzanto) [inline src/chat/transcript/transcript.controller.ts:86]: You should do this stuff in the DTO, not in the controller.

**Review by nitzanto — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (TomerMiz10) [reply src/main.ts:50]: Please change this
